docs/tutorials/build-a-dapp/index.md

```diff
@@ -6,7 +6,14 @@ In this tutorial, you will learn to create a custom
 ["Proof of Existence"](https://en.wikipedia.org/wiki/Proof_of_Existence) dApp using the Substrate
 blockchain development framework and the [FRAME](../../knowledgebase/runtime/frame) library.
 
-This tutorial should take you about **1 hour** to complete. We will be using the
+What is a substrate dApp? The goal, for our purposes, here is a **user facing application**
+that interacts **custom substrate implementation**. The very bare bones required here are:
```

Comment:
This addition seems repetitive to what follows IMO. I like the voice and "hookiness" but I also think that the rhetorical question is out of place here given we assume that readers already have an understanding of what a Substrate dApp is.

Reply:
Sure - happy to remove or have you change language as you see fit 😁